charts/woodpecker/values.yaml

```diff
@@ -40,6 +40,21 @@ agent:
   extraSecretNamesForEnvFrom:
     - woodpecker-secret
 
+  persistence:
+    # -- Enable the creation of the persistent volume
+    enabled: true
+    # -- Defines an existing claim to use
+    existingClaim:
+    # -- Defines the size of the persistent volume
+    size: 1Mi
```

Comment:
is this supposed to be `1Gi`?

Reply:
Why? The volume just holds the agent configuration (text file). `1Mi` should be more than enough as default.

Reply:
it's 1Gi in the agent subchart values (https://github.com/woodpecker-ci/helm/pull/163/files#diff-afab925868b45a9b5ea72b370133adf7a55e74652cfd7695bbeca6b7e2c30658R65) and i think 1Mi is below the minimum of some csi provisioners

Reply:
Ok yes, missed that. Good point in that case we should fix it.